src/AspNet.Security.OAuth.Introspection/OAuthIntrospectionOptions.cs

```diff
@@ -19,7 +20,7 @@ public class OAuthIntrospectionOptions : AuthenticationOptions {
         /// Setting this property is recommended when the authorization
         /// server issues access tokens for multiple distinct resource servers.
         /// </summary>
-        public string Audience { get; set; }
+        public IList<string> Audiences { get; set; }
```

Comment:
Please consider removing the setter and assigning a default instance.
Updating the code documentation to indicate what happens when multiple audiences are set would be great :+1: